backend/app/helpers/spree/admin/base_helper.rb

```diff
@@ -133,16 +133,16 @@ def preference_fields(object, form)
             when :currency
               content_tag(:div, form.label("preferred_#{key}", Spree.t(key)) +
                 (form.select "preferred_#{key}", currency_options(object.preferences[key]), {}, { class: 'form-control select2' }),
-                          class: 'form-group')
+                          class: 'form-group', id: key)
```

Comment:
@MatthewKennedy maybe it would be better to have more unique id eg

```ruby
id: [object.class.parameterize, 'preference', key].join('-')
```

what do you think?

Reply:
That works.

<img width="821" alt="Screenshot 2021-01-14 at 12 00 49" src="https://user-images.githubusercontent.com/1240766/104588499-5eaa4b80-5660-11eb-8d0f-f29d54a23caa.png">

Reply:
Good solid target specific to the payment method.